tests/data/vision/vision_validation.vnf

```diff
@@ -1,131 +1,143 @@
-V9.4
+V9.6
```

Comment:
shouldn't we have both vision versions next to eachother? we still have to keep `V9.4` alive, right?

Reply:
The difference is between 9.6 and 9.7.